package.json

```diff
@@ -11,7 +11,7 @@
   "license": "MIT",
   "devDependencies": {
     "@azure/avocado": "^0.6.3",
-    "@azure/rest-api-specs-scripts": "^0.7.3",
+    "@azure/rest-api-specs-scripts": "^0.8.0",
```

Comment:
When will this version be released to spec repository? Will this version sit in PPE pipeline for long? The reason why I'm asking is I want to have a new patch version change and deploy to PROD pipeline firstly.

Reply:
This version have been applied to PPE pipeline since 4/23 , it's a long time

Reply:
Talked offline with @jianyexi , I will release a new version later after this version release.